warehouse/query-core/src/main/java/datawave/query/jexl/functions/KeyAdjudicator.java

```diff
@@ -46,17 +44,18 @@ public KeyAdjudicator() {
     @Override
     public Entry<Key,T> apply(Entry<Key,T> entry) {
         final Key entryKey = entry.getKey();
-        return Maps.immutableEntry(new Key(entryKey.getRow(), entryKey.getColumnFamily(), colQualRef, entryKey.getColumnVisibility(), entryKey.getTimestamp()),
-                        entry.getValue());
+        if (WaitWindowObserver.hasMarker(entryKey)) {
```

Comment:
I need an explaination as to why we should not be updating the cq in this case.

Reply:
Added comment: if the key has a YIELD_AT_BEGIN or YIELD_AT_END marker, then don't modify the key
because doing so will adversely affect the subsequent yield and re-seek position